### DISCUSSION TITLE
How to use mui libary <FORM> component and sendgrid

### DISCUSSION THREAD

**Discussion by jk0ehn** [neuefische/questions#260] [Web]
Here is my PR
https://github.com/jk0ehn/capstone-project/pull/12

How to declarate my inputs with the Form Component "Box" from the mui?

<img width="501" alt="Bildschirmfoto 2022-10-13 um 10 49 02" src="https://user-images.githubusercontent.com/109575621/195551172-5fbca7bb-100e-43ab-8330-12efbd48ca43.png">

And I also want to use "Sendgrid". I have allready read the docs and created an account and my api, but they use not env. for the API. They use sendgrid.env does this also work?
I had many question to use this. Have somebody experience with that?

**Comment by julialoeschel** [comment]
SendGrid: We dont have any experience here and I would recommend to develop your frontend first. Having a contact form is a good UserStory, collect the data and save them e.g in localstorage first. After  you made that work you can think about implementing an Email Service . 

Please do one step at a time :) 

Your MUI problem is in dicussion. Why do you need mui here?

**Reply by jk0ehn** [reply]
Is there an other way to send the message to a mailadress?

_"Why do you need mui here?"_
I want to use a libary. The icons are from mui too, so why not?
